schema/organization/Organization.md

```diff
@@ -0,0 +1,152 @@
+# Organization
+
+The `Organization` type allows you to provide details about an organization such as its legal name and departments, contact details: address, email and telephone as well as information about its funders. This type of often used to describe the `affiliations` of a [`Person`](/Person).
+
+## Examples
+The examples below are based on the University of Otago and show different ways of specifying an `Organization`. [The University of Otago](https://www.otago.ac.nz/) is the oldest university in Aotearoa New Zealand. 
+
+<!--
+These examples will eventually be wrapped in React components
+to illustrate how the input is converted into Stencila JSON
+See https://github.com/stencila/schema/issues/45
+-->
+
+The University of Otago can be represented in canonical Stencila JSON by:
+
+
+```json
+{
+  "type": "Organization",
+  "address": "362 Leith Street, Dunedin 9054, New Zealand",
+  "brands": "Otago",
+  "contactPoints": {
+    "availableLanguages": ["English", "Māori"],
+    "emails": ["office@otago.ac.nz"],
+    "telephone": "00641234567",
+  },
+  "departments": [Commerce.yaml, Health_Sciences.yaml, Humanities.yaml],
+  "funders": [ MBIE.yaml, CancerSociety.yaml],
+  "legalName": "The University of Otago"
+}
+```
+
+```markdown
+---
+title: Introduction to Computer Science
+authors:
+  - prefix: Dr
+    firstNames: Jane
+    lastName: Smith
+    suffix: PhD
+    affiliations: University of Otago
+      - address: 362 Leith Street, Dunedin 9054, New Zealand
+        brands: Otago
+        contactPoints: 
+           - languages: English, Māori
+             emails: office@otago.ac.nz
+             telephone: 00641234567
+        departments: Department of Computer Science
+        legalName: The University of Otago
+---
```

Comment:
Setting `departments` like this (a) doesn't conform to the schema (it should be a list of `Organization` objects), and (b) implies that "Dr Smith is affiliated with the UoO which only has one department, the Department of Computer Science" whereas the intent, I think is to say "Dr Smith is affiliated with the Department of Computer Science at the UoO". A better way to do this might be to add a [`parentOrganization` property](https://schema.org/parentOrganization) and use that e.g. 

```yaml
    affiliations:
      - type: Organization
        name: Department of Computer Science
        parentOrganization:
          type: Organization
          name: University of Otago
          address: 362 Leith Street, Dunedin 9054, New Zealand
          brands: Otago
          contactPoints: 
            - type: ContactPoint
              languages: English, Māori
              emails: office@otago.ac.nz
              telephone: 00641234567
```